nncf/quantization/fake_quantize.py

```diff
@@ -106,8 +106,9 @@ def tune_range(
         fval = -left_border * s
         qval = fns.round(fval)
 
-    ra = fns.where(qval < level_high, qval / (qval - level_high) * right_border, left_border)
-    rb = fns.where(qval > 0.0, (qval - level_high) / qval * left_border, right_border)
+    with fns.disable_error_handling(qval):
```

Comment:
Please, do not enter such context because user usage scenarios are not clear.

Reply:
removed it, tune_range will be reworked in next pr.